force-app/main/default/aura/contacts/contactsController.js

```diff
@@ -0,0 +1,10 @@
+({
+	handleCreateContact: function(component, event, helper) {
+        var newContact = event.getParam("contactitem");
+        helper.createContact(component, newContact);
+    },
+    handleUpdateContact : function(component, event, helper) {
+        var newContact = event.getParam("contactitem");
```

Comment:
Do we need to create a separate variable as it can be passed directly to helper method